### PR TITLE
Silence 2.7 deprecation warning, pip freeze

### PR DESCRIPTION
**Pull request by sgillies**

### CODE REVIEW

**Comment by rbuffat** [comment]
@sgillies 
Can you try the following?
`  - ps: python -m pip install --no-deps --ignore-installed (gci dist\*.whl | % { "$_" })`

**Comment by sgillies** [comment]
@rbuffat do you have any idea why these Windows builds are failing?

https://ci.appveyor.com/project/sgillies/fiona/builds/28728955/job/xs688eo6lpuxb9jl

I don't see a significant difference to the successful maint-1.9 branch builds (thanks!) but I may be overlooking something.

**Comment by sgillies** [comment]
@rbuffat are you reading my mind? :laughing: You certainly have impeccable timing.

**Comment by rbuffat** [comment]
@sgillies  I just saw the emails before going to bed ;) I think you need also to merge the requirements-dev.txt from maint 1.9

File: appveyor.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ build_script:
 
   # install the wheel
   - ps: python -m pip install --upgrade pip
-  - ps: python -m pip install (gci dist\*.whl | % { "$_" })
+  - ps: python -m pip install -no-deps --ignore-installed (gci dist\*.whl | % { "$_" })
```
Comment (rbuffat) [inline appveyor.yml]: --no-deps

**Comment by rbuffat** [comment]
The last line looks a bit strange. Does python setup.py build_ext bdist_wheel already install Fiona? 

```
python -m pip install --verbose (gci dist\*.whl | % { "$_" })
Created temporary directory: c:\users\appveyor\appdata\local\temp\1\pip-ephem-wheel-cache-g4czx4
Created temporary directory: c:\users\appveyor\appdata\local\temp\1\pip-req-tracker-purokt
Created requirements tracker 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pip-req-tracker-purokt'
Created temporary directory: c:\users\appveyor\appdata\local\temp\1\pip-install-9lzfha
Config variable 'Py_DEBUG' is unset, Python ABI tag may be incorrect
Config variable 'WITH_PYMALLOC' is unset, Python ABI tag may be incorrect
Config variable 'Py_UNICODE_SIZE' is unset, Python ABI tag may be incorrect
Requirement already satisfied: Fiona==1.8.11 from file:///C:/projects/fiona/dist/Fiona-1.8.11-cp27-cp27m-win_amd64.whl in c:\projects\fiona (1.8.11)
```

**Comment by sgillies** [comment]
No, not that I know of @rbuffat. Thanks for your help here. Bon weekend !

**Pull request merged**